content/5.open-source/2.self-hosting.md

```diff
@@ -62,48 +63,42 @@ Here are four different use cases. Please refer to their APP_HOMEPAGE_URL settin
 Run on your own computer
 
 ```yaml [docker-compose.yml]
-version: '3'
+networks:
+  keydb:
+  mongo:
 
 services:
-heyform:
-  image: heyform/community-edition:latest
-  restart: always
-  volumes:
-    # Perist uploaded images
-    - ./assets:/app/static/upload
-  depends_on:
-    - mongo
-    - redis
-  ports:
-    - '9513:8000'
```

Comment:
@Leopere It's about this line, the ports of heyforms. Some people modify this port but mistakenly set `APP_HOMEPAGE_URL `to `http://127.0.0.1:8000`. It should be set to the mapped port, such as `:9513`.